scripts/sites.js

```diff
@@ -99,5 +99,6 @@ const sites = [
   { url: 'https://fundor333.com/', title: 'Fundor333', type: 'blog', author: 'Fundor 333', contact: 'blog@fundor333.com', feed: 'https://fundor333.com/twtxt.txt' },
   { url: 'https://cass.si/', contact: 'm@cass.si' },
   { url: 'https://dotcomboom.somnolescent.net/', title: 'dotcomboom.somnolescent.net', author: 'dcb', contact: 'dotcomboom@somnolescent.net', feed: 'https://dotcomboom.somnolescent.net/twtxt.txt' },
-  { url: 'https://cadmican.neocities.org/', title: 'cadmican', contact: 'arachi@airmail.cc', author: 'Nihiltarier' }
+  { url: 'https://cadmican.neocities.org/', title: 'cadmican', contact: 'arachi@airmail.cc', author: 'Nihiltarier' },
+  { url: 'https://jskjott.com', title: 'jskjott', contact: 'jonathan@jskjott.com', author: 'Jonathan Skjøtt' },
```

Comment:
Could you please remove the trailing comma here?

Reply:
Sure, I just removed the trailing comma ^^